data/strings/types_strings.txt

```diff
@@ -20532,7 +20274,7 @@
     zh-Hant = 買車
 
   [type.shop.car_parts]
-    en = Car Parts
+    en = Car Parts Shop
```

Comment:
I am ok with "Car Parts Shop", but I don't see why Cannabis is Store in the US and Shop in en_GB.   In the US, we say store and shop more or less interchangeably.  I realize that in the UK a place that sells things is almost always shop, vs a place where things are actually stored.

There's a saying in the US by people who only buy things as they need them:  "I store my inventory in the store."  That would likely seem odd to an en_GB speaker.

Reply:
so cannabis should be shop for en-US?

Reply:
Cannabis Store and Cannabis Shop are equally acceptable in the US, and I'd give the edge for Shop.  If it makes it easier to just have en as Shop, that's totally ok.